src/Telegram/Types/Message/Message.php

```diff
@@ -479,6 +480,8 @@ class Message extends BaseType
      */
     public ?SuccessfulPayment $successful_payment = null;
 
+    public ?RefundedPayment $refunded_payment = null;
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common processes. Try to leave a link to the process described so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏On refunded payment listener missing